src/server/job.rs

```diff
@@ -107,6 +112,12 @@ pub struct Job {
 
     pub submission_date: DateTime<Utc>,
     pub completion_date: Option<DateTime<Utc>>,
+
+    /// Holds channels that will receive information about the job task counters after the job
+    /// finishes in any way.
+    ///
+    /// You can subscribe to the finish updates with [`Self::subscribe_to_completion`].
+    completion_handlers: Vec<oneshot::Sender<JobTaskCounters>>,
```

Comment:
Btw: Isnt it more flexible just sent job id rather than just counters? I understand that i would need an extra lookup but it seems to me more flexible and sending counters looks as a kind of hack

Reply:
Currently, there is only one usage that needs job counters, so it could be generalized later. But it's a small thing, I'll change it to job id. The current implementation would also work fine if it returned just `()`.

Reply:
JobId :+1: